### ISSUE TITLE
Roundabouts/Rotaries should include exit numbers and street name

### ISSUE THREAD

**Issue by freenerd** [Project-OSRM/osrm-text-instructions#14]
First, roundabouts should have the same handling as rotaries.

Then, currently roundabouts only cover the first case, but instead should cover all. Let's stay in line with #8 and have all of the string in the instructions.json. Maybe this is the point where we'd need a better templateing language, that can support nesting of attributes, so that we could have a string like this:

`Enter {rotary_name} [=nameandexit and take the {exit} exit onto {way_name}] [=name and exit onto {way_name}] [=exit and take the {exit} exit]`

```
    case 'rotary':
        instruction = instruction.replace('{rotary_name}', step.rotary_name || 'the rotary');
        if (step.name && m.exit) {
            instruction = instruction + ' and take the {exit} exit onto {way_name}';
        } else if (step.name) {
            instruction = instruction + ' and exit onto {way_name}';
        } else if (m.exit) {
            instruction = instruction + ' and take the {exit} exit';
        }
        break;
```

**Comment by freenerd** [comment]
Added roundabout handling being the same as rotary handling in https://github.com/Project-OSRM/osrm-text-instructions/pull/13/commits/9df6319fe57ad5da55958d892cd080993e3c2928 https://github.com/Project-OSRM/osrm-text-instructions/pull/13

Big issue remains #8 here

**Comment by freenerd** [comment]
This has been fixed in #27 with [nested properties](https://github.com/Project-OSRM/osrm-text-instructions/blob/1eae991700536a657004b440640b1c5f352a1f4e/instructions.json#L293-L316) `name`, `exit` and `name_exit`.
